### PR TITLE
Feature/admin edit coupon form

### PR DESCRIPTION
**Pull request by JiMeow**
Not connect with api yet

### CODE REVIEW

File: src/modules/admin/mock/coupon.ts [hunk 1]
```diff
@@ -3,10 +3,12 @@ export type CouponDataType = {
   title: string;
   discount: number;
   point: number;
-  useableMenu: string[];
-  numberOfUsedCoupons: number;
+  redeemableMenu: string[];
+  numberOfReadeamedCoupons: number;
```
Comment (minza55113151) [inline src/modules/admin/mock/coupon.ts:7]: typo

File: src/modules/admin/promotion/console/CouponFormSection.tsx [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+                  if (e.target.value) {
+                    setImageURL(e.target.value);
+                  } else {
+                    setImageURL(
+                      "https://source.unsplash.com/random/?food&plate&11",
+                    );
+                  }
+                }}
+              />
+            </Form.Item>
```
Comment (GolfGrab) [inline src/modules/admin/promotion/console/CouponFormSection.tsx:269]: fix input coupon images

File: src/modules/admin/promotion/editCoupon/CouponDeleteModal.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          status="warning"
+          title={
+            <H4 style={{ paddingBottom: "12px" }}>
+              คุณต้องการจะลบคูปองหรือไม่?
+            </H4>
+          }
+          style={{ paddingBottom: "0px", paddingTop: "0px" }}
+        />
+      </StyledModal>
+    </>
```
Comment (GolfGrab) [inline src/modules/admin/promotion/editCoupon/CouponDeleteModal.tsx:45]: remove extra <> </>

**Review by GolfGrab — APPROVED** [review]
Nice have some Request changes

**Review by GolfGrab — CHANGES_REQUESTED** [review]
Request changes

**Comment by JiMeow** [comment]
I resolved everything but it will in another pull request. So approve me please 😌😌.

**Review by GolfGrab — APPROVED** [review]

**Pull request merged**